src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -50,7 +50,8 @@ public void onInteract(PlayerRightClickEvent e) {
 
             SlimefunItem block = slimefunBlock.get();
 
-            if (block instanceof AbstractAutoCrafter) {
+            // Fixes #2957
+            if (block instanceof AbstractAutoCrafter && !block.isDisabledIn(e.getPlayer().getWorld())) {
```

Comment:
This simple check won't send a message to let the player know what's wrong.
On that occasion, we don't check for unlocked researches either, so this should rather be `.canUse(...)`

Reply:
canUse doesn't check world, should I modify it to do so? That would probably fix even more disabled things working (if disabled for a certain world)

Reply:
It does...
https://github.com/Slimefun/Slimefun4/blob/master/src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java#L1084-L1090